storage-tool.yml

```diff
@@ -310,6 +322,9 @@ services:
       - name: max_file_size
         class: 4 GB
         notes:
+      - name: storage_warmth
+        class: hot
```

Comment:
is lab archives really hot? where is the line between warm and hot?

Reply:
Technically, yes. And I'd actually say that these notebooks are not really for storing "data" in the traditional sense. But our Library friends wanted to add this. They were adamant.